.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
     "eslint.workingDirectories": [
         "./app",
         "./cms"
-    ]
+    ],
+    "workbench.startupEditor": "none",
```

Comment:
didn't seem to work :(    I tested it twice.

Reply:
@Sing-Li I've fixed it now, could you please check now...thank you!

This is actually a bug from VSCode and is still being resolved...I'll be tracking it as well!
https://github.com/microsoft/vscode/issues/58038

And as for the "Gitpod Release Notes" page opening up at start when new user comes in is being tracked and worked over here...though this is just once for new users!
https://github.com/gitpod-io/gitpod/issues/13638
https://github.com/gitpod-io/gitpod/issues/14533

### Detailed Explanation
Due to [this bug](https://github.com/microsoft/vscode/issues/58038) in VSCode of **workspace setting** not overriding the **user settings**, 
If the user manually overrides the settings before ( in your case when you opened Gitpod for the first time..Gitpod automatically sets this value to `"workbench.startupEditor": "welcomePage",` in your **user settings**)

And hence as all this happened before me making changes to the **workspace setting** of `"workbench.startupEditor": "none",`, you'd be experiencing this bug...since **workspace setting** is currently not overriding the **user settings**

SO in conclusion **NEW** gitpod users i.e. without any **user settings** on `"workbench.startupEditor": "..."`, would have the ideal workflow of the website preview and a file like `brandlogo.js` opening side by side!

Reply:
Amazing.  

Cool.  Tested.  Agreed this is the best we can do for now.